frontend/src/components/pod/__snapshots__/Pod.stories.storyshot

```diff
@@ -212,8 +213,15 @@ exports[`Storyshots pods/PodListRenderer Pods 1`] = `
               </td>
               <td
                 class="MuiTableCell-root MuiTableCell-body"
+                style="text-align: right;"
               >
-                just now
+                <p
+                  class="MuiTypography-root makeStyles-noWrap MuiTypography-body1"
+                  title="4/25/2022, 12:00:00 AM UTC"
+                >
+                  3mo
```

Comment:
I wonder if this will be "3mo" next month? I don't think this test will be stable.

ps. TIL: I wasn't entirely sure "mo" was widely understood as an abbreviation for month. After doing some googling... a number of ranking pages do seem to put it first.

Reply:
Good point. I will fix this in a different PR.

Reply:
It's fixed in #630